documentation/src/main/asciidoc/stories/assembly_marshalling_user_types.adoc

```diff
@@ -0,0 +1,16 @@
+[id='marshalling_user_types']
+:context: marshalling
+= Marshalling User Types with ProtoStream
+User types are Java objects that {brandname} does not support out of the box.
+To marshall user types, you implement the `SerializationContextInitializer`
+interface to describe your Java objects so that the ProtoStream library can
+encode them to Protobuf format and {brandname} can transmit and store them.
+
+include::{topics}/proc_generating_protostream_sci.adoc[leveloffset=+1]
+include::{topics}/proc_manually_registering_sci.adoc[leveloffset=+2]
+include::{topics}/proc_implementing_protostream_sci.adoc[leveloffset=+1]
+include::{topics}/proc_adding_deserialization_allowlist.adoc[leveloffset=+1]
```

Comment:
This isn't required for protostream, so this should just be linked from the docs of Marshallers that require it, e.g. `JavaSerializationMarshaller` or `JbossUserMarshaller`.

Reply:
ah... OK. I thought it was required for all cases where user objects were used. will fix.